src/NuGet.Core/NuGet.ProjectModel/IncludeExcludeFiles.cs

```diff
@@ -81,10 +86,10 @@ public bool Equals(IncludeExcludeFiles other)
                 return true;
             }
 
-            return Include.SequenceEqualWithNullCheck(Include) &&
```

Comment:
I think this type falls under https://github.com/NuGet/Home/issues/9149, I'll make sure to include it when I handle that issue.